CHANGES.md

```diff
@@ -1,5 +1,25 @@
 # Changes
 
+## edge-22.10.1
+
+This edge release fixes some sections of the Viz dashboard that were appearing
```

Comment:
i think that we can drop the "that were" here to make this a little more concise?
```suggestion
This edge release fixes some sections of the Viz dashboard appearing
```